composer.json

```diff
@@ -24,13 +24,13 @@
         "predis/predis"  : "v0.8.5",
         "guzzle/guzzle": ">=3.8@stable",
         "doctrine/common": "^2.5",
-        "talis/persona-php-client": ">=3.1"
+        "talis/persona-php-client": "^4.0.0"
```

Comment:
Yes. But don't we need to have a version of all the client libraries with the same version of the persona client in them? Using ^4.0.0 means that they could pick up different versions at the point in time you generate the composer.lock. You'd have to go and check the composer.lock files to ensure they all had the same version in them.

I think it's better to lock onto a specific version to ensure it's that version being used consistently between the different clients.